newsplease/pipeline/extractor/article_extractor.py

```diff
@@ -35,15 +36,19 @@ def proc_instance(instance):
             else:
                 extractor_module = extractor
 
-            module = importlib.import_module(__package__ + '.extractors.' + extractor_module)
+            module_name = __package__ + '.extractors.' + extractor_module
+            module = importlib.import_module(module_name)
 
             if isinstance(extractor, tuple):
                 proc_instance(getattr(module, extractor[1], None)())
             else:
-                # check module for subclasses of AbstractExtractor
+                # check in the current module for subclasses of AbstractExtractor
                 for member in inspect.getmembers(module, inspect.isclass):
-                    if issubclass(member[1], AbstractExtractor) and member[0] != 'AbstractExtractor':
-
+                    if (
+                        member[1].__module__ == module_name
```

Comment:
So we filter out `NewspaperExtractor` which is not in the file newspaper_extractor_no_images.py